react-app/src/components/forms/SendTokenForm/SendTokenFormModel.ts

```diff
@@ -0,0 +1,47 @@
+import { useMemo } from "react";
+import BigNumber from "bignumber.js";
+
+import { RegisterOptions } from "react-hook-form";
+import { useFormValidators } from "../../../hooks/useFormValidators";
+import { useLocale } from "../../../providers/AppLocaleProvider";
+
+export interface SendTokenFormValues {
+  recipent: string;
+  amount: string;
+  memo: string | null;
+}
+
+export const useSendTokenFormModel = (
+  availableTokens: BigNumber
+): {
+  registerOptions: Record<keyof SendTokenFormValues, RegisterOptions>;
+} => {
+  const { address, tokenAmount, memo } = useFormValidators();
+  const { translate } = useLocale();
+  const registerOptions = useMemo((): Record<
+    keyof SendTokenFormValues,
+    RegisterOptions
+  > => {
+    return {
+      recipent: address,
```

Comment:
can we name the exported validators xxxValidator so it is more readable when imported & used?